widget/playground/src/components/Slider/Slider.tsx

```diff
@@ -19,8 +20,21 @@ function Slider(props: PropTypes) {
     variant = 'custom',
     min,
     max,
+    id,
   } = props;
 
+  const progressScript = () => {
```

Comment:
Good job. Shouldn't we include dependencies in the useEffect?

Reply:
Yes, it is included in the useEffect